packages/sdk/client-services/src/packlets/invitations/invitations-service.ts

```diff
@@ -31,6 +34,8 @@ export class InvitationsServiceImpl implements InvitationsService {
   constructor(
     private readonly _invitationsHandler: InvitationsHandler,
     private readonly _getHandler: (invitation: Invitation) => InvitationProtocol,
+    // TODO(nf): avoid making InvitationManager?
```

Comment:
what do you mean?

Reply:
that the service is just a passthrough for the storage and doesn't have any state or logic of its own.